cookiecutter/prompt.py

```diff
@@ -17,21 +17,88 @@
 from jinja2.environment import Environment
 
 
-def read_response(prompt=''):
-    """Prompt the user and return the entered value or an empty string.
+def read_user_variable(var_name, default_value):
+    """Prompt the user for the given variable and return the entered value
+    or the given default.
 
-    :param str prompt: Text to display to the user
+    :param str var_name: Variable of the context to query the user
+    :param default_value: Value that will be returned if no input happens
+    """
+    # Please see http://click.pocoo.org/4/api/#click.prompt
+    return click.prompt(var_name, default=default_value)
+
+
+def read_user_yes_no(question, default_value):
+    """Prompt the user to reply with 'yes' or 'no' (or equivalent values).
+
+    Note:
+      Possible choices are 'true', '1', 'yes', 'y' or 'false', '0', 'no', 'n'
+
+    :param str question: Question to the user
+    :param default_value: Value that will be returned if no input happens
     """
     # Please see http://click.pocoo.org/4/api/#click.prompt
-    # default: use an empty string if no input happens
-    # prompt_suffix: do not add a suffix to the prompt
-    # show_default: the prompt is expected to mention the default itself
     return click.prompt(
-        prompt,
-        default='',
-        prompt_suffix='',
-        show_default=False,
+        question,
+        default=default_value,
+        type=click.BOOL
+    )
+
+
+def read_user_choice(var_name, options):
+    """Prompt the user to choose from several options for the given variable.
+
+    The first item will be returned if no input happens.
+
+    :param str var_name: Variable as specified in the context
+    :param list options: Sequence of options that are available to select from
+    :return: Exactly one item of ``options`` that has been chosen by the user
+    """
+    # Please see http://click.pocoo.org/4/api/#click.prompt
+    if not isinstance(options, list):
+        raise TypeError
+
+    if not options:
+        raise ValueError
+
+    if not all(is_string(opt) for opt in options):
```

Comment:
This is actually a mistake on my part. `click` requires the actual choices to be strings however I am not using them directly but generate numbers on the fly.

Removed this part as well as the corresponding tests.

:bow: